source/frontend/src/features/mapSideBar/research/tabs/fileDetails/update/UpdateSummaryForm.tsx

```diff
@@ -162,11 +162,13 @@ const UpdateSummaryForm: React.FunctionComponent<IUpdateSummaryFormProps> = prop
             }}
           />
         </SectionField>
-        {values.requestor?.id.startsWith('P') && (
-          <SectionField label="Organization" className="pb-4">
-            {values.requestor.organizationName ?? 'none'}
-          </SectionField>
-        )}
+        {values.requestor &&
+          isPersonResult(values.requestor) &&
+          values.requestor.person?.personOrganizations?.length !== undefined && (
```

Comment:
not sure this guarantees there is at least one personorganization given 0 is different than undefined